### PR TITLE
scala and sbt update

### PR DESCRIPTION
**Pull request by sideeffffect**

### CODE REVIEW

File: project/BuildHelper.scala [hunk 1]
```diff
@@ -145,7 +145,7 @@ object BuildHelper {
   def stdSettings(prjName: String) = Seq(
     name := s"$prjName",
     scalacOptions := stdOptions,
-    crossScalaVersions := Seq("2.12.9", "2.13.0", "2.11.12"),
+    crossScalaVersions := Seq("2.12.9", "2.13.1", "2.11.12"),
```
Comment (regiskuckaertz) [inline project/BuildHelper.scala]: is there anything holding off upgrading to 2.12.10 at the same time?

File: sbt [hunk 2]
```diff
@@ -6,24 +6,24 @@
...
 
-declare -r sbt_release_version="0.13.16"
-declare -r sbt_unreleased_version="0.13.16"
+declare -r sbt_release_version="1.3.2"
+declare -r sbt_unreleased_version="1.3.2"
 
-declare -r latest_213="2.13.0-M2"
-declare -r latest_212="2.12.4"
+declare -r latest_213="2.13.1"
+declare -r latest_212="2.12.10"
```
Comment (regiskuckaertz) [inline sbt]: ha, great!

Comment (sideeffffect) [reply sbt]: this comes with upgrading the `sbt-extras` script

**Comment by sideeffffect** [comment]
blocked on https://github.com/ghik/silencer/issues/35

**Comment by regiskuckaertz** [comment]
Woops, there's duplicated work on #1701

**Comment by sideeffffect** [comment]
@regiskuckaertz 
actually not 100% duplicate, so both PRs can be merged, once silencer for 2.13.1 is released

**Comment by ghostdogpr** [comment]
@sideeffffect silencer was released: https://search.maven.org/search?q=a:silencer-plugin_2.13.1 can you rebase and update?

File: project/BuildHelper.scala [hunk 3]
```diff
@@ -161,15 +160,18 @@ object BuildHelper {
...
-    crossScalaVersions := Seq("2.12.9", "2.13.0", "2.11.12"),
+    crossScalaVersions := Seq("2.12.10", "2.13.1", "2.11.12"),
     scalaVersion in ThisBuild := crossScalaVersions.value.head,
     scalacOptions := stdOptions ++ extraOptions(scalaVersion.value, optimize = !isSnapshot.value),
-    libraryDependencies ++= compileOnlyDeps ++ testDeps,
+    libraryDependencies ++= testDeps,
     libraryDependencies ++= {
       if (isDotty.value)
-        Seq()
+        Seq("com.github.ghik" % "silencer-lib_2.13.1" % "1.4.4" % Provided)
```
Comment (sideeffffect) [inline project/BuildHelper.scala]: @ghostdogpr are we happy with this hack?

Comment (ghostdogpr) [reply project/BuildHelper.scala]: No other way I guess? I’d say it’s fine since this plugin has very limited impact.

**Pull request merged**